build.gradle

```diff
@@ -118,6 +118,9 @@ subprojects {
         compileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombok
         annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
         testAnnotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+
+        implementation 'org.springframework.boot:spring-boot-starter-validation'
```

Comment:
Please try by removing this dependency

Reply:
Certain annotations like NotNull depends on this package. So this dependency is required